web/src/unread.ts

```diff
@@ -1028,6 +1039,10 @@ export function get_msg_ids_for_user_ids_string(user_ids_string: string): number
     return unread_direct_message_counter.get_msg_ids_for_user_ids_string(user_ids_string);
 }
 
+export function message_has_mention(message_id: number): boolean {
+    return direct_message_with_mention_count.has(message_id);
+}
```

Comment:
I'd prefer that this helper not exist. Its name is misleading, since it only works for direct messages, but nowhere indicates that without reading its code. The use in this file, at least, seems like it would be better off just accessing `direct_message_with_mention_count` directly.